frontend/integration-tests/tests/crud.scenario.ts

```diff
@@ -295,20 +314,19 @@ describe('Kubernetes resource CRUD operations', () => {
     beforeAll(async() => {
       await browser.get(`${appHost}/k8s/ns/${testName}/${plural}/new`);
       await yamlView.isLoaded();
-
       const content = await yamlView.editorContent.getText();
-      const newContent = _.defaultsDeep({}, {metadata: {name, namespace: testName}}, safeLoad(content));
-      await yamlView.setContent(safeDump(newContent));
+      const newContent = _.defaultsDeep({}, {metadata: {name, namespace: testName}}, await safeLoad(content));
+      await yamlView.setContent(await safeDump(newContent));
```

Comment:
Aren't `safeLoad` and `safeDump` synchronous? I wouldn't expect to need `await` here